Changes.md

```diff
@@ -13,6 +13,11 @@ Fixes
 - Outputs : Custom `gaffer:renderID` parameters are now preserved, allowing custom "netrender" style workflows to be implemented.
 - RenderPassEditor : Fixed bug that could cause duplicate tabs to appear when registering custom columns to an existing section via a different group key.
 
+API
+---
+
+- Metadata : The `registerNode()` function now accepts dictionaries containing plug metadata. This hsould be preferred to the previous list-based values.
```

Comment:
hsould -> should

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/6615/commits/3078fcd3d310a46309f05489db6ecf621fd656cf, where I also fixed the Changes.md merge conflict.